app/services/generate_transaction_file.service.js

```diff
@@ -4,54 +4,81 @@
  * @module GenerateTransactionFileService
  */
 
-const fs = require('fs')
-const path = require('path')
-const { Readable } = require('stream')
-const { temporaryFilePath } = require('../../config/server.config')
+const { TransactionModel } = require('../models')
+const TransformRecordsToFileService = require('./transform_records_to_file.service')
+const {
+  TransactionFileBodyPresenter,
+  TransactionFileHeadPresenter,
+  TransactionFileTailPresenter
+} = require('../presenters')
 
 class GenerateTransactionFileService {
   /**
-   * Generates and writes a transaction file to a given filename in the temp folder.
+   * Generates and writes a transaction file for a bill run to a given filename in the temp folder.
```

Comment:
```suggestion
   * Generates and writes a transaction file for a bill run to a given filename in the /tmp folder.
```

Reply:
Rightly pointed out by @StuAA78 . You can override what the temp folder is using the config so it is better to refer to it as `temp` rather than being specific and using `/tmp`